docs/conf/modules.conf.example

```diff
@@ -1232,8 +1232,13 @@
 # Closes the channel for N seconds if X users join in Y seconds.
 #<module name="joinflood">
 #
-# The number of seconds to close the channel for:
```

Comment:
You should retain the description of 'duration'. Probably best to reword it though, as it can be a time string not just number of seconds.

Reply:
Fixed